.gitignore

```diff
@@ -40,3 +40,4 @@ custom_plan.rb
 pkg/
 config/hooks
 *.csv
+.envrc
```

Comment:
It's related - it's the configuration file for [direnv](https://github.com/direnv/direnv/#readme) which spring recommends as a good way to add ./bin/ to your PATH automatically when you're in the Rails project directory, saving you from typing it out.

In not unrelated news, direnv RPMs for Fedora are in [my copr](https://copr.fedorainfracloud.org/coprs/domcleal/direnv/).